js/renovation/ui/form/form.tsx

```diff
@@ -7,15 +7,40 @@ import { FormProps } from './form_props';
 
 import { combineClasses } from '../../utils/combine_classes';
 import { Widget } from '../common/widget';
+import { Scrollable } from '../scroll_view/scrollable';
 
 export const viewFunction = (viewModel: Form): JSX.Element => {
-  const { aria, cssClasses, restAttributes } = viewModel;
-  return (
-    <Widget
-      aria={aria}
-      classes={cssClasses}
-      {...restAttributes} // eslint-disable-line react/jsx-props-no-spreading
-    />
+  const {
+    aria,
+    cssClasses,
+    props: {
+      scrollingEnabled,
+      useNativeScrolling,
+    },
+    restAttributes,
+  } = viewModel;
+  return (scrollingEnabled
+    ? (
+      <Scrollable
+        aria={aria}
```

Comment:
Why? Was it in the 'old' code?

Reply:
Form added it to Scrollable: https://github.com/DevExpress/DevExtreme/blob/21_1/js/ui/form/ui.form.js#L315